app/javascript/packages/webauthn/webauthn-input-element.ts

```diff
@@ -15,11 +19,13 @@ export class WebauthnInputElement extends HTMLElement {
   }
 
   async toggleVisibleIfPasskeySupported() {
+    const webauthnPlatformAvailable = await isWebauthnPlatformAuthenticatorAvailable();
+
     if (!this.hasAttribute('hidden')) {
       return;
     }
 
```

Comment:
Minor: If we have an early return, I think we can avoid any extra work until we're certain to get past that early return, especially when it's asynchronous and might take a while.

```suggestion
    if (!this.hasAttribute('hidden')) {
      return;
    }

    const webauthnPlatformAvailable = await isWebauthnPlatformAuthenticatorAvailable();
```